docs/modules/ROOT/pages/usage.adoc

```diff
@@ -29,7 +30,36 @@ Both Hive and Trino need the same S3 authentication.
 
 Please refer to the https://github.com/stackabletech/opa-operator[OPA] operator and https://docs.stackable.tech/home/index.html[docs].
```

Comment:
This is not sufficient. An opa cluster needs to be deployed, that should be added to the documentation. Also, the link to the docs just points to the index, it should point to the opa docs.